Engine/IocContainer.cpp

```diff
@@ -126,32 +126,32 @@ Vis *IocContainer::vis = nullptr;
 
 void IntegrityTest() {
     // ToDo(_): make platform independent
-    static_assert(sizeof(BLVSector) == 0x74, "Wrong type size");
```

Comment:
is fread/memcpy on all of those addressed?

Reply:
No, I will need to go through them all except 1-2 that came up during early testing.

Reply:
I checked and most are not being filled directly with fread / memcpy. I left those commented out. For the others I have created _MM7 structs, constructors and serializers.